app/views/posts/_post.html.erb

```diff
@@ -13,10 +13,36 @@
 <p class="post-time-ago my-2"><%= time_ago_in_words(post.created_at) %> ago.</p>
 <p class="post-content"><%= post.content %></p>
 <div class="d-flex justify-content-between post-like-line">
-	<button class="trans-btn"><i class="fa fa-thumbs-o-up" aria-hidden="true"></i>  Like</button>
+	<ul class="d-flex pl-0">
+		<li>
+			<% unless current_user.likes.exists?(post_id: post.id) %>
+			  <%= button_to "👍",{:controller => 'likes', :action => 'create',:user_id => current_user.id, :post_id => post.id}, {:method => :post, :class => "trans-btn font-red mid-font"} %>
+			<% else %>
+			  <%= button_to "👎",{:action => 'destroy',
+			  	:controller => 'likes',:id => post.likes.find_by(user_id:current_user.id).id }, 
```

Comment:
**[REQUIRED]**
Let's move all the view related logic to the helpers. Methods in `app/helpers` are automatically available to the views.
([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/section/no_logic_in_helpers_requirement.png) from Pathwright).

**More Resources** 
* [Where Do I Put My Code?](http://codefol.io/posts/Where-Do-I-Put-My-Code).
* [Beginners Guide to Rails helpers](https://mixandgo.com/learn/the-beginners-guide-to-rails-helpers)

Reply:
We also put the view related logic `post.likes.find_by(user_id:current_user.id).id }` into a helper method